src/tox/session/cmd/version_flag.py

```diff
@@ -5,17 +5,28 @@
 
 from pathlib import Path
 
+import tox
 from tox.config.cli.parser import ToxParser
 from tox.plugin import impl
+from tox.plugin.manager import MANAGER
+from tox.version import version
 
 
 @impl
 def tox_add_option(parser: ToxParser) -> None:
-    import tox
-    from tox.version import version
```

Comment:
Was there a reason those where imported locally and not in the import section?

Reply:
Perhaps circular import issues 🤔

Reply:
I was not familiar with those issues, I ran [pycycle](https://pypi.org/project/pycycle/) and looks like we're fine with them in the import section.